.github/workflows/continuous_delivery.yml

```
@@ -1,39 +1,48 @@
-
+    branches: [ "main", "dev", "feature/refactor" ] # Added dev branch
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
     permissions:
       contents: read
-      packages: write   # allow pushing to GHCR (GitHub Packages) 
+      packages: write # allow pushing to GHCR (GitHub Packages)
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)